cmd/charm/charmcmd/publish.go

```diff
@@ -105,6 +110,8 @@ func (c *publishCommand) Run(ctxt *cmd.Context) error {
 	if err != nil {
 		return errgo.Notef(err, "cannot publish charm or bundle")
 	}
+	fmt.Fprintln(ctxt.Stdout, "url:", c.id)
```

Comment:
If the revision needs to be specified, this really is only printing what we specified on the command line, so meh.

If we really need to print something, perhaps:

fmt.Fprintf(ctxt.Stdout, "%s published to channel %s\n", c.id, c.channel)

?